dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -543,3 +596,32 @@ describe('Clean up all test data', () => {
   });
 });
 
+describe('Verify Operational Panels Table', () =>{
+  it('Add Dummy Data to Operational Panel and verify Table Column, Pagination and Rows Data ', () => {
+  var i = 0;
+  for (i = 0; i < 12 ; i++) { 
+  createDummyPanel(); }
+  cy.get('.euiTableCellContent__text').contains('Name').should('exist');
+  cy.get('.euiTableCellContent__text').contains('Last updated').should('exist');
+  cy.get('.euiTableCellContent__text').contains('Created').should('exist');
+  moveToPanelHome();
+  cy.get('[data-test-subj="pagination-button-next"]').click();
+  cy.get('.euiTableCellContent--truncateText').contains('TestPanel1').should('exist');
+  cy.get('[data-test-subj="pagination-button-previous"]').click();
+  cy.wait(delay);
+  cy.get('.euiTableCellContent--truncateText').contains('TestPanel12').should('exist');
+  cy.get('.euiButtonEmpty__text').contains('Rows per page').click();
+  cy.get('.euiContextMenuItem__text').contains('8 rows').click();
+  let expected_row_count=8;
+  cy.get('.euiTable--auto')
+  .find("tr")
+  .then((row) => {
+    let total=row.length-1;
```

Comment:
Inconsistent spaces around `=`. Check the complete file for this and fix them please.

Reply:
Done

Reply:
I do not see a space